packages/tokens/config.js

```diff
@@ -0,0 +1,66 @@
+// eslint-disable-next-line no-undef
+const StyleDictionary = require("style-dictionary");
+const EDSStyleDictionary = StyleDictionary.extend({
+  source: ["properties/**/*.json"],
+  platforms: {
+    scss: {
+      transforms: [...StyleDictionary.transformGroup.scss, "name/kebabCase"],
+      buildPath: "dist/scss/",
+      files: [
+        {
+          destination: "_variables.scss",
+          format: "scss/map-deep",
+        },
+      ],
+    },
+    css: {
+      transforms: [...StyleDictionary.transformGroup.css, "name/kebabCase"],
+      buildPath: "dist/css/",
+      files: [
+        {
+          format: "css/variables",
+          destination: "variables.css",
+        },
+      ],
+    },
+    js: {
+      transformGroup: "js",
+      buildPath: "dist/js/",
+      files: [
+        {
+          format: "javascript/es6",
+          destination: "colors.js",
+          filter: {
+            attributes: {
+              type: "color",
+            },
+          },
+        },
+      ],
+    },
+    json: {
+      transformGroup: "js",
+      buildPath: "dist/json/",
+      files: [
+        {
+          format: "json/flat",
+          destination: "variables.json",
+        },
+        {
+          format: "json/nested",
+          destination: "variables-nested.json",
+        },
+      ],
+    },
+  },
+});
+
+EDSStyleDictionary.registerTransform({
+  name: "name/kebabCase",
+  type: "name",
+  transformer: function ({ path }) {
+    return path.join("-");
+  },
+});
+
```

Comment:
This makes sure the heading tokens are output as `size-font-h1` instead of split into `size-font-h-1`. This is an open issue for style-dictionary: https://github.com/amzn/style-dictionary/issues/300, so I used the solution in thread. Looks like they might address it at some point, but haven't yet

Reply:
should we maybe to `path.join('-').toLowerCase()`?